apps/dashboard/app/views/projects/show.html.erb

```diff
@@ -10,4 +10,39 @@
   </p>
 </div>
 
+<div class='card'>
+  <h3 class='card-header'>Scripts</h3>
+  <div class='card-body'>
+    <table class='table table-borderless'>
+      <thead>
+        <tr>
+          <th>Script Name</th>
+          <th>Actions</th>
+        <tr>
+      </thead>
+      <tbody>
+      <% @scripts.each do |script| %>
+        <tr>
+          <td><%= script.title %></td>
+          <td>
+            <%= link_to(
+                  'Show',
+                  project_script_path(@project.directory, script.id),
+                  class: 'btn btn-success'
+                )
+            %>
+          <td>
+        <tr>
+      <% end %>
+      </tbody>
+    </table>
+
+    <%= link_to('New Script', 
+          new_project_script_path(@project.directory),
+          class: 'btn btn-info',  
+          title: I18n.t('dashboard.jobs_project_create_new_project_directory'))
+    %>
+  </div>
+</div>
```

Comment:
I'm kiinda confused why a `view` is in the PR...

Reply:
There's no way to write correct software without something exercising it. If I just wrote the model, without anything that can _exercise it_, there'd be no way to _verify_ it.

That is, what I have here may be slightly buggy, but does work. Just writing the model is unlikely to work at all. I'd be guessing at what we need, and worse still, would probably write code that doesn't work at all due to simple mistakes.

Reply:
Interesting. One could just spec out what data the model needs at a fundamental level and write tests to makes sure all works, but I think I see what you mean.

We can pull these in as the PR is in fact vague and doesn't really say models are all that are being worked on.